### DISCUSSION TITLE
Running into problem with Prisma when using Git submodules

### DISCUSSION THREAD

**Discussion by M4TY** [prisma/prisma#14463] [Q&A]
My current folder structure is:

```
repo 1:

//npm files//
src/
    core (repo 2)

repo 2:
//npm files//
prisma/
src/
     controllers/
```

when I call a function from _core_ (submodule), and I have some Prisma logic there, it throws this error:
`Error: ENOENT: no such file or directory, open '/Users/_xx_/_xx_/_xx_/repo1/node_modules/.prisma/client/schema.prisma'
`

I understand that it is trying to find the prisma dependencies, am I able to redirect it to the submodule, as I don't want to have Prisma as a dependency on repo 1?

**Comment by austincrim** [comment]
Hey @M4TY 👋,

Could you share a minimal reproduction of your setup? This will help out a lot while I try to reproduce your exact error.

**Reply by M4TY** [reply]
Hey @austincrim, 

this is my submodule code: 
```ts
//other imports
import {client} from "../../utils/prismaClientWrapper";
async function createUser(data: IUser) {
    let dbData = client.user.findMany();
    return true;
}

export default createUser;
```

and in the repo that the submodule is in, I have this piece of code:

```ts
import express from "express";
import createUser from "../core/src/controllers/users/createUser";
import {response} from "../utils/response";
import {Debugger} from "inspector";
const router = express.Router();

router.post("/new", async (req: express.Request, res: express.Response) => {
    //validate body data
    if(await createUser(req.body)) {
        response(res, 200, "User created");
    } else {
        response(res, 400, "Bad request");
    }
});
router.get("/", () => console.log("New user"));

export const users = router;
```

This is prismaClientWrapper.ts (which is in the submodule)
```ts
import { PrismaClient } from '@prisma/client'

let p = new PrismaClient();
p.$connect();

export const client = p;
```

**Reply by austincrim** [reply]
Could you create a GitHub repository that mirrors your setup, including folder structure?

**Reply by M4TY** [reply]
I have invited you!

**Reply by M4TY** [reply]
Hey @austincrim, have you managed to replicate the issue?

**Reply by pantharshit00** [reply]
Hey @M4TY 

Austin has left the team. Can you please invite me instead?

**Reply by M4TY** [reply]
Hello @pantharshit00, I have already figured it out. Thanks for the effort tho!

**Reply by vladi-stevanovic** [reply]
Hello @M4TY thank you for sharing this is resolved now, that's great news! When you have a moment could you share the solution? It would be very helpful for any new users who face the same issue! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
